src/components/toast-provider.tsx

```
@@ -1,18 +1,61 @@
 import type { ReactNode } from 'react'
+import type { ToastUpdateEvent } from '@/utils/toast'
+import { useEventBus } from '@hairy/react-lib'
 import { Toast } from '@heroui/react'
-import { activeQueues, placements } from '@/utils/toast'
+import { useState } from 'react'
+import { If } from 'react-if-lite'
+import { placements, activeQueues } from '@/utils/toast'
```

ESLint

[error] Expected "activeQueues" to come before "placements".

(perfectionist/sort-named-imports)